chainerio/_context.py

```diff
@@ -62,55 +57,52 @@ def is_supported_scheme(self, scheme: str) -> bool:
 class DefaultContext(object):
     def __init__(self):
         self._fs_handler_list = FileSystemDriverList()
-        self._root = ""
 
         self._default_context = \
-            self._fs_handler_list.get_handler_for_root("posix")[0]
+            self._fs_handler_list.get_handler("posix")[0]
+
+        self._global_handler_cache = {}
```

Comment:
We know `DefaultContext` is global object so the name of this object could be just `_handler_cache` or simply `_handlers` . Even if any of these names adopted, what is stored in this dict is not clear so it must be documented in the comment ("global handler cache" isn't even enough from the viewpoint of readability) .

Reply:
resolved in 228f0ef

Reply:
resolved in ba72759